### PR TITLE
Support for host mode

### PR DESCRIPTION
**Pull request by gbarros**
I basically followed the pattern already available at the Sunodo template for TS.

### CODE REVIEW

File: packages/app/src/app.ts [hunk 1]
```diff
@@ -12,6 +12,9 @@ import {
     InspectRequestData,
 } from "./types";
 
+type RollupsRequest = components["schemas"]["RollupRequest"];
+
+
```
Comment (tuler) [inline packages/app/src/app.ts:17]: remove this extra line

File: packages/app/src/app.ts [hunk 2]
```diff
@@ -139,7 +143,9 @@ export class AppImpl implements App {
                         );
                         break;
                 }
-            }
+            } else if(response.status == 202) {
+                console.log(await response.text());
```
Comment (tuler) [inline packages/app/src/app.ts:147]: I think I'd rather not include a `console.log` here, as this is a library.

In the future we may have some proper `logger` infrastructure, then we add a log

File: packages/app/src/app.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 import createClient from "openapi-fetch";
 
-import { paths } from "./schema";
+import { paths, components} from "./schema";
```
Comment (tuler) [inline packages/app/src/app.ts:3]: sorry for being pick, we still have to include the prettier check in the CI.
There is a missing space here.

Comment (gbarros) [reply packages/app/src/app.ts:3]: No worries 😅 😆

**Pull request merged**